client/src/app/pages/issues/issue-detail-drawer/file-incidents-detail-modal/incident-code-snip-viewer.tsx

```diff
@@ -42,6 +52,25 @@ export const IncidentCodeSnipViewer: React.FC<IIncidentCodeSnipViewerProps> = ({
       ] as Language)
     : undefined;
 
+  if (
+    incident.codeSnip === undefined ||
```

Comment:
If `incident.codeSnip` really could be `undefined` or `null`, this entire empty detection block should happen before any of the other parsing code.

Line 29 would fail before getting here if  `codeSnip` is `undefined` or `null`.